tools-local/tasks/core-setup.tasks.csproj

```diff
@@ -34,6 +34,9 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' != 'net45'">
     <Compile Include="netstandard/FileUtilities.netstandard.cs" />
+
+    <Reference Include="$(BuildToolsTaskCoreDir)Microsoft.DotNet.Build.CloudTestTasks.dll" />
```

Comment:
Can you use `$(BuildToolsTaskDir)` instead?  That way you can put these two lines into a common spot.